### PR TITLE
Add Kettle device

### PR DESCRIPTION
**Pull request by skyfall174**

### CODE REVIEW

File: src/main/kotlin/onm/house/devices/Kettle.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import onm.events.IEventHandler
+import onm.events.WasherDoneEvent
+import java.util.*
+
+class Kettle(override val id: UUID, eventHandler: IEventHandler) : AbstractDevice(DeviceType.KETTLE) {
+
+    private val event = WasherDoneEvent(eventHandler)
+
+    /**
+     * Starts washing clothes. This produces event which is raised after given time period. Note that new thread is created.
```
Comment (LukasForst) [inline src/main/kotlin/onm/house/devices/Kettle.kt:13]: That does not start washing clothes.

File: src/main/kotlin/onm/house/devices/Kettle.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+     * */
+    fun startWarmingWater(periodInMinutes: Double) {
```
Comment (LukasForst) [inline src/main/kotlin/onm/house/devices/Kettle.kt:15]: I personally think that kettle does not get time period... it just notifies user when watter is ready

**Review by LukasForst — CHANGES_REQUESTED** [review]

File: src/main/kotlin/onm/house/devices/Kettle.kt (hunk 2 above)
Comment (HappyStoic) [reply src/main/kotlin/onm/house/devices/Kettle.kt:15]: Same here. Just startWarmingWater(){ ... }. Btw pull master, merge and add please kettle to HouseBuilder

**Review by HappyStoic — CHANGES_REQUESTED** [review]
viz comment

**Pull request closed without merging**